src/sections/create-lead-view/schema.ts

```diff
@@ -21,17 +23,148 @@ export type Shape<Fields> = {
   [Key in keyof Fields]: ConditionalSchema<Fields[Key]>;
 };
 
+const RANGES = {
+  totalAreaMin: 0,
+  totalAreaMax: 10000,
+  priceRangeRentMin: 100,
+  priceRangeRentMax: 10000,
+  priceRangeBuyMin: 10000,
+  priceRangeBuyMax: 10000000,
+};
```

Comment:
if you have this numbers if why you don't reuse it?

Reply:
fixed